app/Services/Sync.php

```diff
@@ -77,9 +83,18 @@ public static function actionSingleQueueItem($cli = false) {
       // time before retries
       // max number of tries
 
-      $worker->pop('default', getenv('AWS_SQS_CATFISH_IMPORTER_QUEUE'), 0, 3, 0);
+      $return = $worker->pop('default', getenv('AWS_SQS_CATFISH_IMPORTER_QUEUE'), 0, 3, 0);
+
+      // Queue item ran successfully, ping the Envoyer heartbeat URL to stay we're still alive
+      file_get_contents(getenv('ENVOYER_HEARTBEAT_URL_IMPORTER'));
```

Comment:
Can you throw an Exception "ENVOYER_HEARTBEAT_URL_IMPORTER missing from .env" if the env value is not present before doing the file_get_contents on it please?